src/main/java/org/folio/services/validator/registry/ValidatorRegistryServiceImpl.java

```diff
@@ -175,7 +175,7 @@ private Criteria constructCriteria(String jsonbField, String value) {
     Criteria criteria = new Criteria();
     criteria.addField(jsonbField);
     criteria.setOperation("=");
-    criteria.setValue(value);
+    criteria.setVal(value);
```

Comment:
You can drop this constructCriteria method if you use the PostgresClient methods that directly take the id.
id and ruleId are always the same.

Reply:
Can you add the "id" property to rule.json, rule.sample, ruleCollection.sample?